javatests/arcs/core/entity/HandleManagerCloseTest.kt

```diff
@@ -0,0 +1,76 @@
+package arcs.core.entity
+
+import arcs.core.data.HandleMode
+import arcs.core.data.Ttl
+import arcs.core.entity.HandleManagerTestBase.Hat
+import arcs.core.entity.HandleManagerTestBase.Person
+import arcs.core.host.EntityHandleManager
+import arcs.core.storage.StorageKey
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import arcs.core.storage.keys.RamDiskStorageKey
+import arcs.core.storage.referencemode.ReferenceModeStorageKey
+import arcs.jvm.host.JvmSchedulerProvider
+import arcs.jvm.util.testutil.FakeTime
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Before
+import org.junit.Test
+import kotlin.coroutines.EmptyCoroutineContext
+
+class HandleManagerCloseTest {
+
+    val scheduler = JvmSchedulerProvider(EmptyCoroutineContext).invoke("test")
+
+    private val backingKey = RamDiskStorageKey("entities")
+    private val singletonRefKey = RamDiskStorageKey("single-ent")
+    private val singletonKey = ReferenceModeStorageKey(
+        backingKey = backingKey,
+        storageKey = singletonRefKey
+    )
+
+    @Before
+    fun setup() {
+        DriverAndKeyConfigurator.configure(null)
+        SchemaRegistry.register(Person)
+        SchemaRegistry.register(Hat)
+    }
+
```

Comment:
Should you add a test here to see if an exception is thrown if you try to use something already closed?

Reply:
good idea